packages/test-cypress/cypress/e2e/tagSpecific/answer.cy.js

```diff
@@ -38,15 +38,23 @@ describe("Answer Tag Tests", function () {
             let mathInputPartialAnchor = cesc("#" + mathInputName + "_partial");
 
             // cy.get(mathInputAnchor).should('have.value', '');
-            cy.get(mathInputSubmitAnchor).should("be.visible");
+            cy.get(mathInputSubmitAnchor).should(
+                "have.attr",
+                "aria-label",
+                "Check Work",
+            );
```

Comment:
You should be able to do this in a non-cypress test, but maybe it is easiest to put it here?